test/elements.js

```diff
@@ -104,6 +104,53 @@ test('adjacent text nodes', function (t) {
   t.end()
 })
 
+test('space in only-child text nodes', function (t) {
+  t.plan(1)
+  var result = bel`
+    <span>
+      surrounding
+      newlines
+    </span>
+  `
+  t.equal(result.outerHTML, '<span>surrounding\n      newlines</span>', 'should remove extra space')
```

Comment:
Perhaps my intuition of how HTML works is wrong, but wouldn't this be closer to what folks might expect:
```js
 t.equal(result.outerHTML, '<span>surrounding newlines</span>', 'should remove extra space')
```